EarlGrey/Action/GREYActions.m

```diff
@@ -330,6 +330,16 @@ + (void)grey_webSetText:(id)element text:(NSString *)text {
       [GREYActions grey_webSetText:element text:text];
     } else {
       [element setText:text];
+            // For UIControl-s, send actions for UIControlEventEditingChanged, as it is not sent when directly
+      // setting the text.
+      [element sendActionsForControlEvents:UIControlEventEditingChanged];
+      // For UITextField-s, send the UITextFieldTextDidChangeNotification notification, as it is not sent
```

Comment:
I think the code is cleaner without these comments since they just describe what the code is doing.